specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,392 @@
+              $ref: '#/components/schemas/Woonplaats'
+    GbaVerblijfplaatsBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
+            straat:
+              $ref: '#/components/schemas/Straat'
+            huisnummer:
+              $ref: '#/components/schemas/Huisnummer'
+            huisletter:
+              $ref: '#/components/schemas/Huisletter'
+            huisnummertoevoeging:
+              $ref: '#/components/schemas/Huisnummertoevoeging'
+            aanduidingBijHuisnummer:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            postcode:
+              $ref: '#/components/schemas/Postcode'
+            locatiebeschrijving:
+              $ref: '#/components/schemas/Locatiebeschrijving'
+            land:
+              $ref: 'land.yaml#/components/schemas/Land'
+            gemeenteVanInschrijving:
+              $ref: '#/components/schemas/GemeenteVanInschrijving'
+            naamOpenbareRuimte:
+              $ref: '#/components/schemas/NaamOpenbareRuimte'
+            adresregel1:
+              $ref: '#/components/schemas/Adresregel1'
+            adresregel2:
+              $ref: '#/components/schemas/Adresregel2'
+            adresregel3:
+              $ref: '#/components/schemas/Adresregel3'
+    AbstractVerblijfplaats:
+      type: object
+      description: |
+        Gegevens over het verblijf of de woonlocatie van een persoon.
+      required:
+        - type
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaVerblijfplaats: '#/components/schemas/GbaVerblijfplaats'
+          VerblijfplaatsBuitenland: '#/components/schemas/VerblijfplaatsBuitenland'
+          Verblijfplaats: '#/components/schemas/Verblijfplaats'
+          VerblijfOnbekend: '#/components/schemas/VerblijfplaatsOnbekend'
+          Locatie: '#/components/schemas/Locatie'
+    VerblijfplaatsBuitenland:
+      allOf:
+      - $ref : '#/components/schemas/AbstractVerblijfplaats'
+      - type: object
+        properties:
+          adresregel1:
+            $ref: '#/components/schemas/Adresregel1'
+          adresregel2:
+            $ref: '#/components/schemas/Adresregel2'
+          adresregel3:
+            $ref: '#/components/schemas/Adresregel3'
+          land:
+            $ref: 'Land.yaml#/components/schemas/AbstractLand'
+          datumAanvangAdreshouding:
+            $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+          datumIngangGeldigheid:
+            $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+          inOnderzoek:
+            $ref: '#/components/schemas/VerblijfplaatsBuitenlandInOnderzoek'
+    Verblijfplaats:
+      description: |
+        Gegevens over het verblijf of de woonlocatie van een persoon.
```

Comment:
de omschrijving klopt niet meer. Het gaat niet over een woonlocatie, maar om een adres. Woonlocatie zit in component Locatie.

Reply:
aangepast